Microsoft.Identity.Web/MsalUiRequiredExceptionFilterAttribute.cs

```diff
@@ -16,15 +19,22 @@ namespace Microsoft.Identity.Web.Client
     /// The following controller action will trigger
     /// <code>
     /// [MsalUiRequiredExceptionFilter(Scopes = new[] {"Mail.Send"})]
-    /// public async Task<IActionResult> SendEmail()
+    /// public async Task&lt;IActionResult&gt; SendEmail()
     /// {
     /// }
     /// </code>
     /// </example>
     public class MsalUiRequiredExceptionFilterAttribute : ExceptionFilterAttribute
```

Comment:
>MsalUiRequiredExceptionFilterAttribute [](start = 17, length = 38)

Learning from the existing AuthorizeAttribute in use in as.net and asp.net core, the attribute traps the AccessDeniedException and acts on it. This attribute is somewhat similar in that respect. Can we consider renaming this to MsalAuthorizeAttribute or MsalAuthorizeForScopesAttribute or something similar .IMHO, using an exception in an attribute name like we have for  MsalUiRequiredExceptionFilterAttribute, carries somewhat negative connotations.

Reply:
I like MsalAuthorizeForScopesAttribute or AuthorizeForScopesAttribute
What about EnsureScopesAttribute?

Reply:
I renamed to AuthorizeForScopesAttribute.